### PR TITLE
Implement web search on a given host

### PR DESCRIPTION
**Pull request by caggle**

### CODE REVIEW

**Comment by caggle** [comment]
Ready for review.

This one is pretty straightforward - implement an on-demand endpoint which performs an web search on a host submitted.

Scheduled search is not implemented because I don't see it adding any value (this one should come in particularly useful for first time 3rd parties).

File: tests/test_websearcher.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        # Stub env var for testing
+        host_name = "www.mozilla.org"
+        web_searcher = WebSearcher()
+        search_results = web_searcher.search(host_name)
+
+        # Check to see is some of the expected JSON elements
+        # are in the result to ensure it's working
+        assert 'host' in search_results
+        assert 'search' in search_results
+        assert len(search_results['search']) <= 15
```
Comment (claudijd) [inline tests/test_websearcher.py]: I think you should have a lower bound here as well.  If it's zero (and potentially broken), this will still pass.

Comment (caggle) [reply tests/test_websearcher.py]: Good point, will add.

Comment (caggle) [reply tests/test_websearcher.py]: Added another check. Note this will dismiss your review, but if CI passes (it should) I will just admin-merge it.

**Pull request merged**